### DISCUSSION TITLE
Don't open cart after product add

### DISCUSSION THREAD

**Discussion by Daxpnx** [bootscore/bootscore#373] [Q&A]
Hi,

In WooCommerce theme, after click to "add to cart" button, I don't want to open the cart (on the right side).
Can you help me ?

Regards
Damien

**Comment by justinkruit** [comment]
Please provide a link to the website so we can debug the problem! 😊

**Comment by Daxpnx** [comment]
You can see on this page : https://lejardinduperecocq.fr/reserver-un-panier/

**Comment by crftwrk** [comment]
As I understand, you want to keep offcanvas cart closed if a product is added to the cart, right? There are 3 ways to do this:

1. Disable ajax functionality general as described here https://bootscore.me/documentation/woocommerce/#Disable_ajax_cart. You should choose this because there are cart issues with your configuration in product pages https://lejardinduperecocq.fr/produit/physalis/.
2. Remove offcanvas in `header.php` and link cart toggler button in navbar to the cart page.
3. Override `ajax-add-to-cart.php` in your child and remove offcanvas open snippet.

Think point 1 and 2 are clear. If you choose way 3, do this:

Go to main-theme's folder `woocommerce/ajax-cart/` and copy `ajax-add-to-cart.php` to your child-theme. Use same path as in main-theme by creating folders `woocommerce/ajax-cart/` in child. Path should be `woocommerce/ajax-cart/ajax-add-to-cart.php`.

Register new `ajax-add-to-cart.php` by adding this snippet to your child's `functions.php`:

```php
// Register new ajax-add-to-cart.php in child
function register_ajax_cart() {
  require_once('woocommerce/ajax-cart/ajax-add-to-cart.php');
}
add_action('after_setup_theme', 'register_ajax_cart');
```

Now, new file is used and you can edit it without touching the main-theme.

Open child's `ajax-add-to-cart.php`, there you will find this:

https://github.com/bootscore/bootscore/blob/d18b9163303f0122ad39c5651838b35265906771/woocommerce/ajax-cart/ajax-add-to-cart.php#L160-L163

Comment or delete snippet and offcanvas cart doesn't open anymore if product has been added to the cart. But you should keep an eye on this file, because there will be changes in future.

Does this help?

**Comment by Daxpnx** [comment]
Yes! thank you for your help !  
I will try one of these solutions ;)
